app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -16,7 +18,9 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void test(int a) {
-        if (a > 3)return;
+        if (a > 3){
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L21

&apos;{&apos; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*